tests/components/bond/test_light.py

```diff
@@ -27,12 +27,17 @@ def ceiling_fan(name: str):
     }
 
 
+def fireplace(name: str):
+    """Create a fireplace with given name."""
+    return {"name": name, "type": DeviceTypes.FIREPLACE}
+
+
 async def test_entity_registry(hass: core.HomeAssistant):
     """Tests that the devices are registered in the entity registry."""
     await setup_platform(hass, LIGHT_DOMAIN, ceiling_fan("name-1"))
 
     registry: EntityRegistry = await hass.helpers.entity_registry.async_get_registry()
-    assert [key for key in registry.entities.keys()] == ["light.name_1"]
+    assert [key for key in registry.entities] == ["light.name_1"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)